pages/events/index.vue

```diff
@@ -62,15 +62,18 @@
           </div>
         </template>
         <template #results>
-          <UiCard
-            v-if="noEvents"
-            :image="emptyCard.img"
-            :title="emptyCard.title"
-          >
-            <div class="event-page__empty-card-description">
-              {{ emptyCard.description }}
+          <div v-if="noEvents" class="bx--row">
+            <div class="cds--col-sm-4 cds--col-xlg-8">
+              <UiCard
```

Comment:
```suggestion
              <UiCard :image="emptyCard.img" :title="emptyCard.title">
```

To get rid of the lint error (in this particular case), we should place the component props on one line